build/tutorials/tutorials-contest/hardhat-fork/test/swap.ts

```diff
@@ -0,0 +1,76 @@
+import { ethers } from "hardhat";
+import * as dotenv from "dotenv";
+import { SignerWithAddress } from "hardhat-deploy-ethers/src/signers";
+import { BigNumber } from "ethers";
+import { Swapper, IWAVAX } from "../typechain";
+import { expect } from "chai";
+
+dotenv.config();
+
+const WAVAX_ADDRESS: string = process.env.WAVAX_ADDRESS as string;
+const AVALANCHE_NODE_URL: string = process.env.AVALANCHE_MAINNET_URL as string;
+const PNG_ADDRESS = "0x60781C2586D68229fde47564546784ab3fACA982"
+
+describe("Swappity swap", function () {
```

Comment:
Use fat arrow functions and add return types for functions.

Reply:
Done